src/Game.cpp

```diff
@@ -0,0 +1,85 @@
+//
+// Created by Ruoying Hao on 10/17/17.
+//
+
+#include "../include/Game.hpp"
```

Comment:
I think it should be okay to just say Game.hpp?

Reply:
True